src/pages/account/profile/user-profile/new.tsx

```diff
@@ -1,18 +1,49 @@
+import { HttpStatus } from "@fwl/web";
```

Comment:
@Fenntasy Can I have your output on this? I just realised that we do the same in there https://github.com/fewlinesco/connect-account/blob/main/src/pages/account/profile/index.tsx

My only issue is that we need to add a client side fetch to get the info.